azure-iot-hub-health-server/server.ts

```diff
@@ -0,0 +1,154 @@
+namespace jacdac {
+    type SMap<T> = { [s: string]: T }
+    function splitPair(kv: string): string[] {
+        let i = kv.indexOf("=")
+        if (i < 0) return [kv, ""]
+        else return [kv.slice(0, i), kv.slice(i + 1)]
+    }
+
+    function parsePropertyBag(msg: string, separator?: string): SMap<string> {
```

Comment:
probably should export that (maybe with leading underscore) from azureiot?

Reply:
Yes. I have a todo for that. Also disconnect.